samples/bluetooth/hci_lpuart/prj.conf

```diff
@@ -1,5 +1,6 @@
 CONFIG_SERIAL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_NRFX_GPIOTE_NUM_OF_EVT_HANDLERS=2
```

Comment:
Since this setting is now required by the lpuart driver, perhaps it would be better instead to set this value as default when the driver is enabled (and document this requirement)?

Reply:
The default value can be overridden in the application config if needed.

Reply:
done